From: Jakub Jirutka <jakub@jirutka.cz>
Date: Fri, 21 Jul 2023 13:26:43 +0200
Subject: [PATCH] Fix test integration::node_compat_tests::node_compat_tests

1. `await dnsPromises.resolveSoa('come.on.afhqwhgads.test')` does not throw
   ENOTFOUND as expected, but resolves to:

    ```json
    {
      nsname: "come.on.afhqwhgads.test",
      hostmaster: "nobody.invalid",
      serial: 1,
      refresh: 3600,
      retry: 1200,
      expire: 604800,
      minttl: 10800
    }
    ```

2. blog.nodejs.org is not a CNAME anymore.

--- a/cli/tests/node_compat/test/common/internet.js
+++ b/cli/tests/node_compat/test/common/internet.js
@@ -30,7 +30,7 @@
   MX_HOST: 'nodejs.org',
   // On some systems, .invalid returns a server failure/try again rather than
   // record not found. Use this to guarantee record not found.
-  NOT_FOUND: 'come.on.fhqwhgads.test',
+  NOT_FOUND: 'does-not-exist.example.org',
   // A host with SRV records registered
   // TODO(kt3k): Temporarily use _caldav._tcp.google.com instead of
   // _jabber._tcp.google.com, which currently doesn't respond
@@ -45,7 +45,7 @@
   // A host with CAA record registered
   CAA_HOST: 'google.com',
   // A host with CNAME records registered
-  CNAME_HOST: 'blog.nodejs.org',
+  CNAME_HOST: 'pkgs.alpinelinux.org',
   // A host with NS records registered
   NS_HOST: 'nodejs.org',
   // A host with TXT records registered
